src/logic.js

```diff
@@ -12,7 +12,13 @@ define([
             "dataParent",
             "repeat_count",
             "filter",
-            "defaultValue"
+            "defaultValue",
+        ],
+        LABEL_REFERENCES = [
+            "helpItext",
+            "constraintMsgItext",
+            "labelItext",
+            "hintItext",
```

Comment:
Interesting. I wonder if we should add an API for plugins to register property names that may contain logic references? As is this couples the logic manager to javaRosa. Not a blocker, but something to think about.